### PR TITLE
Add pages delegate

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

File: Source/PagesController.swift [hunk 1]
```diff
@@ -107,6 +120,11 @@ extension PagesController {
     }
   }
 
+  public func pageViewController(pageViewController: UIPageViewController, didFinishAnimating finished: Bool, previousViewControllers: [AnyObject], transitionCompleted completed: Bool) {
```
Comment (kostiakoval) [inline Source/PagesController.swift]: every parameter on new line style ?

Comment (vadymmarkov) [inline Source/PagesController.swift]: @kostiakoval I guess in style guidelines we agreed on one line break for functions with long signatures, didn't we? I mean I could add line break, but don't think it's important to place every parameter on the new line.

Comment (3lvis) [inline Source/PagesController.swift]: Might be better to break it for consistency, unless you have a proposal to make us :D

File: Source/PagesController.swift [hunk 2]
```diff
@@ -45,7 +52,13 @@ extension PagesController {
       currentIndex = index
       setViewControllers([viewController],
         direction: direction,
-        animated: true, completion: nil)
+        animated: true,
+        completion: {
+          [unowned self] (finish: Bool) -> Void in
```
Comment (3lvis) [inline Source/PagesController.swift]: I'm still learning Swift but I was thinking if the `-> Void` is needed. Can you do something like this? :

``` swift
completion: { [unowned self] finished in
  self.pagesDelegate?.pageViewController(
    self,
    setViewController: viewController,
    atPage: currentIndex)
})
```

Comment (vadymmarkov) [inline Source/PagesController.swift]: Agree, it looks much better.

**Pull request merged**